pkg/resizer/csi_resizer.go

```diff
@@ -107,7 +110,18 @@ func (r *csiResizer) Name() string {
 	return r.name
 }
 
-func (r *csiResizer) CanSupport(pv *v1.PersistentVolume) bool {
+// CanSupport returns whether the PV is supported by resizer
+// Resizer will resize the volume if it is CSI volume or is migration enabled in-tree volume
+func (r *csiResizer) CanSupport(pv *v1.PersistentVolume, pvc *v1.PersistentVolumeClaim) bool {
+	resizerName := pvc.Annotations[util.VolumeResizerKey]
+	// resizerName will be CSI driver name when CSI migration is enabled
```

Comment:
This is more of a question for in-tree, but does the annotation get updated if we turn on migration after the PVC has already been annotated with the in-tree name?

Reply:
It does resets the annotation if migration has been enabled after annotation was already set.

Reply:
Good question! @gnufied do you know the answer to this?